src/Illuminate/Foundation/helpers.php

```diff
@@ -542,13 +542,13 @@ function method_field($method)
     /**
      * Send to users a notification instance.
      *
-     * @param $users
-     * @param $notification
-     * @return mixed
+     * @param  $users
+     * @param  $notification
+     * @return void
      */
     function notify($users, $notification)
     {
-        return app(Notificantion::class)->send($users, $notification);
+        app(Notificantion::class)->send($users, $notification);
```

Comment:
'Notification' is spelled wrong here and where it is pulled in at the top of this file.